dist/templates/register.eta

```diff
@@ -1,21 +1,22 @@
 <% layout('./layout') %>
-<p>Hi <%= it.member.name %>,</p>
-<p>Thank you for signing up to our platform.</p>
-<p>Sometimes you just want to send a simple HTML email with a simple design and clear call to action. This is it.</p>
-<table role="presentation" border="0" cellpadding="0" cellspacing="0" class="btn btn-primary">
-  <tbody>
-    <tr>
-      <td align="left">
-        <table role="presentation" border="0" cellpadding="0" cellspacing="0">
-          <tbody>
-            <tr>
-              <td> <a href="<%= it.link %>" target="_blank">Complete registration and sign in</a> </td>
-            </tr>
-          </tbody>
-        </table>
-      </td>
-    </tr>
-  </tbody>
-</table>
-<p>This is a really simple email template. Its sole purpose is to get the recipient to click the button with no distractions.</p>
-<p>Good luck! Hope it works.</p>
+  <p>Hi <%= it.member.name %>,</p>
+  <p>Thank you for signing up to our platform.</p>
+  <p>Please verify your email by clicking on the following button.</p>
+
+  <table role="presentation" border="0" cellpadding="0" cellspacing="0" class="btn btn-primary">
+    <tbody>
+      <tr>
+        <td align="center">
+          <table role="presentation" border="0" cellpadding="0" cellspacing="0">
+            <tbody>
+              <tr>
+                <td> <a href="<%= it.link %>" target="_blank">Complete registration and sign in</a> </td>
+              </tr>
+            </tbody>
+          </table>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+
+  <p>If you don't remember registering on Graasp, you can just ignore this email.</p>
```

Comment:
Make sure you add the extra lines (should be in an `.editorconfig`) to avoid GitHub complaining.

Reply:
Change "registering on" to "signing up to".